<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsForm</class>
 <widget class="QWidget" name="SettingsForm">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="toolTip">
      <string/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="3" column="0">
          <widget class="QLabel" name="generalDateFormatLabel">
           <property name="text">
            <string>General date format</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="generalDateFormatLineEdit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="transactionTableDateFormatLabel">
           <property name="text">
            <string>Transaction Table date format</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="transactionTableDateFormatLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="exchangeRateDecimalsLabel">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of decimal places of Exchange Rate spinbox when manually setting Exchange Rate quote in Currencies Form.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Exchange Rate decimals</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="exchangeRateDecimalsSpinBox">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of decimal places of Exchange Rate spinbox when manually setting Exchange Rate quote in Currencies Form.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="pricePerShareDecimalsLabel">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of decimal places of Amount per Share spinbox in Security Transaction Dialog.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Amount per Share decimals</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="pricePerShareDecimalsSpinBox">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of decimal places of Amount per Share spinbox in Security Transaction Dialog.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="checkForUpdatesLabel">
           <property name="text">
            <string>Check for updates on startup</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="checkforUpdatesCheckBox">
           <property name="text">
            <string/>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="numberFormatLabel">
           <property name="text">
            <string>Number format</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="numberFormatComboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;For details on valid date format syntax, see &lt;a href=&quot;https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;Python datetime library documentation&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="backupsTab">
      <attribute name="title">
       <string>Backups</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QFormLayout" name="backupsFormLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="backupsSizeLimitLabel">
           <property name="text">
            <string>Maximum backup directory size</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="backupsSizeLimitSpinBox">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;1 MB = 1,000 kB&lt;br/&gt;1 GB = 1,000,000 kB&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="showGroupSeparator" stdset="0">
            <bool>true</bool>
           </property>
           <property name="suffix">
            <string> kB</string>
           </property>
           <property name="maximum">
            <number>1000000000</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="backupDirectoriesLabel">
         <property name="text">
          <string>Backup directories:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="backupsListView"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="backupsButtonHorizontalLayout">
         <item>
          <widget class="QPushButton" name="addBackupDirectoryButton">
           <property name="text">
            <string>Add directory</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeBackupDirectoryButton">
           <property name="text">
            <string>Remove directory</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="openBackupDirectoryButton">
           <property name="text">
            <string>Open directory in File Browser</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="backupsLine">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="backupsDisclaimerLabel">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;NOTE: A backup file is created in all backup directories whenever a file is opened or saved. Kapytal automatically deletes the contents of a backup directory, starting from the oldest file, if the size limit is exceeded. To keep any of the backups permanently, copy them elsewhere or set a very large size limit. Do not store any other data in the backup directories.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignJustify|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="logsTab">
      <attribute name="title">
       <string>Logging</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="logsFormLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="logsSizeLimitLabel">
           <property name="text">
            <string>Maximum logs directory size</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="logsSizeLimitSpinBox">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;1 MB = 1,000 kB&lt;br/&gt;1 GB = 1,000,000 kB&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="showGroupSeparator" stdset="0">
            <bool>true</bool>
           </property>
           <property name="suffix">
            <string> kB</string>
           </property>
           <property name="maximum">
            <number>1000000000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="openLogsDirectoryButton">
         <property name="text">
          <string>Open logs directory in File Browser</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="logsLine">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="logsDisclaimerLabel">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;NOTE: Logs contain records of all operations performed by Kapytal. A new log is created each time the program is launched. Logs contain potentially sensitive data entered into Kapytal by the user. The logs exist only locally and are not uploaded anywhere. The purpose of the logs is to provide debugging information if unexpected behaviour or an error occurs. Upon launch, Kapytal automatically deletes the oldest logs until the size limit is satisfied. Do not store any other data in the logs directories.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignJustify|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="logsVerticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
